### PR TITLE
Prepare next releases

### PR DESCRIPTION
**Pull request by jderusse**
component | version
----------|--------
core | 1.19
appsync | 1.1.0
athena | 1.0.0
cloud-formation | 1.4.0
cloud-front | 1.0.0
codebuild | 1.0.0
cloud-watch-log | 1.6.0
codebuild | 1.0.0
comprehend | 1.0.0
dynamodb | 1.4.0
ecr | 1.4.0
elasti-cache | 1.1.0
event-bridge | 1.4.1
iot | 1.0.0
iot-data | 1.0.0
kinesis | 2.0.0
kms | 1.0.0
lambda | 1.9.0
rekognition | 1.0.0
route53 | 1.1.0
s3 | 1.14.0
scheduler | 1.0.0
secrets-manager | 1.0.0
sqs | 1.8.0
ssm | 1.4.0
step-functions | 1.1.0
timestream-query | 1.0.0
timestream-write | 1.1.0
symfony-bundle | 1.11.0
monolog-cloud-watch | 1.0.1
xray | 1.1.0

### CODE REVIEW

File: src/Service/Kinesis/CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,15 @@
 
 ## NOT RELEASED
 
+## 2.0.0
+
+### BC-BREAK
+
+- BC BREAK: `listStreams` now yield over stream names AND descriptions
```
Comment (jderusse) [inline src/Service/Kinesis/CHANGELOG.md:9]: Am I right here? 

For reference, this has been introduced by AWS in https://github.com/async-aws/kinesis/commit/0841292426cadbe269a54f102b750935f77219da

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]
async-aws/timestream-query 1.0.0 is missing from this list

**Comment by stof** [comment]
it would be great to merge #1417 first too, so that it could be released as well.

File: src/Core/CHANGELOG.md [hunk 2]
```diff
@@ -2,9 +2,13 @@
 
 ## NOT RELEASED
 
+## 1.19
```
Comment (stof) [inline src/Core/CHANGELOG.md:5]: should be `1.19.0` for consistency.

File: src/Core/CHANGELOG.md [hunk 3]
```diff
@@ -2,9 +2,13 @@
...
+
 ### Added
 
-- Support for Athena
+- Added support for Symfony 7
+- Added support for Athena
+- Added support for IMDS v2 authentication
 - Support for using endpoint discovery with parameters passed in the query string or the path
```
Comment (stof) [inline src/Core/CHANGELOG.md:12]: should `Added` be added here as well ?

File: src/Core/CHANGELOG.md [hunk 4]
```diff
@@ -2,9 +2,13 @@
 
 ## NOT RELEASED
 
+## 1.19
+
 ### Added
 
-- Support for Athena
```
Comment (stof) [inline src/Core/CHANGELOG.md:7]: this was actually consistent with many previous versions.

**Comment by stof** [comment]
@jderusse as our `BC checks` CI jobs are currently broken, maybe they should be fixed before the release ? See the discussions on Slack about that topic.

**Comment by jderusse** [comment]
> @jderusse should MediaConvert start as 0.1.0 or should it go directly to 1.0.0 ?

We usually start with a `0.1` version to allow the contributor to test it and see if they need more operations. 
If you believe this is not necessary we can skip this and directly start at `1.0`. Just tell me what's your preference.

**Comment by jderusse** [comment]
BC check still fails.. I've no clue how this thing work / should work
https://github.com/async-aws/comprehend/actions/runs/5253712955/jobs/9491391212

Any idea?

**Comment by GrahamCampbell** [comment]
It is because you need to add `fetch-depth: 0` when checking out as per the README: https://github.com/actions/checkout#fetch-all-history-for-all-tags-and-branches.

**Pull request merged**